core/pl/util.pm.sdoc

```diff
@@ -90,3 +90,8 @@ sub wf {
   close $fh;
   $_[0];
 }
+
+sub testpath {
+  $_ =~ s/-\*/-0000\*/;
```

Comment:
`$_ =~ ...` == `...`, so you can golf this a bit -- also `sr()` may be what you want here.

This is a totally useful but kinda specific function -- should it go into `core/hadoop/...`?